app/attributes/shopify/bundled_product_attributes.rb

```diff
@@ -0,0 +1,48 @@
+module Shopify
+  class BundledProductAttributes
+    include Spree::Retail::PresenterHelper
+
+    def initialize(bundle, product_converter: Shopify::BundleConverter,
+                   part_attributes: Shopify::PartAttributes,
+                   permuter: Spree::Retail::BundlePermuter)
+
+      @bundle = bundle
+      @converter = product_converter
+      @part_attributes = part_attributes
+      @permuter = permuter
```

Comment:
It takes all the possible option_values of the bundle and it creates all the different permutation. 
Let's say we have the following case instance:
{ { BlueGloss, RedGloss, PinkGloss }
{ RedLip, PurpleLip, PinkLip } }

the outcome of this will be the permutation of every scenario, which would output an array with 9 possible permutation.